modules/wordads/php/widgets.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * Widget for inserting an ad into your sidebar
+ *
+ * @since 0.1
+ * TODO: sanitize all the output
+ */
+class WordAds_Sidebar_Widget extends WP_Widget {
+
+	private static $allowed_tags = array( 'mrec', 'wideskyscraper' );
+
+	private $options = array();
+
+	function __construct() {
+		parent::__construct(
+			'wordads_sidebar_widget',
+			__( 'WordAds Widget (Jetpack)', 'jetpack' ),
```

Comment:
It might be best to use the `jetpack_widget_name` filter here to add `(Jetpack)`